Player Movement & Input System Completed

marytheproducer opened the pull request.

marytheproducer committed:
Smooth Move & Look based around Cinemachine

marytheproducer committed:
Smooth Jumping & Input Component Added

marytheproducer committed:
Crouching Done

marytheproducer committed:
Merge remote-tracking branch 'origin/dev' into PlayerMovement

MJE10 commented on EEG Game/Assets/Scripts/Input/InputManager.cs:
```diff
@@ -0,0 +1,55 @@
+
+    private void Awake()
+    {
+        // Make InputMager a Singleton
```
"InputMager" typo?

marytheproducer replied:
Oh my bad let me fix that also yee I'll look into mouse sensitivity real quick

marytheproducer replied:
I think we can keep the dragonslayer assets for testing

MJE10 reviewed (changes requested):
See possible typo in InputManager.cs
- Mouse sensitivity seems extremely high, hard to control where I look
- There are still some assets with the DragonSlayer naming in Assets/Content, is the plan to rename or remove these at some point or no?

marytheproducer committed:
Fix Typo & Added Camera Sensitivity

Now Camera Sensitivity is based on Input Value Gain which should make the camera feel alot smoother.

The actual setting of camera sensitivity can now be done through the inputmanager making it easier to test.

MJE10 reviewed (approved):
That feels much better

MJE10 commented:
PR moved to #3
